### PR TITLE
Remove usage of deprecated getLogFile

### PR DESCRIPTION
**Pull request by fredrikand**
Resolve JENKINS-54840

### CODE REVIEW

**Comment by timja** [comment]
@rsandell mind taking a look please?

also @fredrikand your git config doesn't look right on your commit
https://help.github.com/en/articles/setting-your-commit-email-address-in-git

File: src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/MultilineBuildLogFailureReader.java [hunk 1]
```diff
@@ -93,7 +96,7 @@ public FoundIndication scan(Run build, PrintStream buildLog) {
         BufferedReader reader = null;
         long start = System.currentTimeMillis();
         try {
-            reader = new BufferedReader(build.getLogReader());
+            reader = new BufferedReader(new InputStreamReader(build.getLogInputStream(), Charsets.UTF_8));
```
Comment (TWestling) [inline src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/MultilineBuildLogFailureReader.java]: Same as above.

File: src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/MultilineBuildLogFailureReader.java [hunk 2]
```diff
@@ -65,7 +68,7 @@ public FoundIndication scan(Run build) throws IOException {
         String currentfile = build.getLogFile().getName();
         BufferedReader reader = null;
         try {
-            reader = new BufferedReader(build.getLogReader());
+            reader = new BufferedReader(new InputStreamReader(build.getLogInputStream(), Charsets.UTF_8));
```
[the remark this answers is not in the document]
Comment (fredrikand) [reply src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/MultilineBuildLogFailureReader.java]: I tried to find a suggestion to apply to the change, and give you cudos for your finding. Either my GitHub skills are lacking or I just managed to loose the suggestions. Will force push with your changes.

File: src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/MultilineBuildLogFailureReader.java [hunk 3]
```diff
@@ -65,7 +66,7 @@ public FoundIndication scan(Run build) throws IOException {
         String currentfile = build.getLogFile().getName();
         BufferedReader reader = null;
         try {
-            reader = new BufferedReader(build.getLogReader());
+            reader = new BufferedReader(new InputStreamReader(build.getLogInputStream(), build.getCharset()));
```
Comment (timja) [inline src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/MultilineBuildLogFailureReader.java]: getLogReader isn't deprecated just getLogFile
I've opened another PR for it:
https://github.com/jenkinsci/build-failure-analyzer-plugin/pull/104

**Comment by rsandell** [comment]
I this superceded by #104 ?

**Comment by fredrikand** [comment]
Yes, or possibly #106.

**Pull request closed without merging**